backend/typescript/models/scheduling.model.ts

```diff
@@ -85,6 +86,11 @@ export default class Scheduling extends Model {
   @Column({ type: DataType.INTEGER })
   donor_id!: number;
 
+  @ForeignKey(() => Volunteer)
+  @AllowNull(true)
+  @Column({ type: DataType.INTEGER })
+  volunteer_id!: number;
+
```

Comment:
I think this does not define the foreign key. You will need to add this just like how it is defined for the Donor: 
```suggestion
@BelongsTo(() => Volunteer)
  volunteer!: Volunteer;
```